app/views/part/test.scala.html

```diff
@@ -106,14 +108,17 @@
         <fieldset>
             <legend>@Messages("tryIt.meta")</legend>
             @for(metaProp <- metaProps) {
-                <div class="form-group">
-                    <label for="meta.@metaProp" class="col-sm-2 control-label">@metaProp</label>
-                    <div class="row">
-                        <div class="col-sm-5">
-                            <input name="meta.@metaProp" id="meta.@metaProp" type="text" class="form-control" />
+                @defining(part.registeredParamsView.find(_.inputNameJs == s"meta.$metaProp")) { metaParam =>
+                    <div class="form-group">
+                        <label for="meta.@metaProp" class="col-sm-2 control-label">@metaProp</label>
+                        <div class="row">
+                            <div class="col-sm-5">
+                                <input name="meta.@metaProp" id="meta.@metaProp" type="text" class="form-control @{metaParam.fold("optional")(_.requiredCls)}"/>
```

Comment:
`requiredCls` is probably something that should be declared here in this view (it's only used here) as a defined as a local function or as helper inside the `view` package.

Right now, it's declared inside a presentation class and then we have to to funky stuff like `fold("optional")` which somewhat defeats the purpose of it. Having the CSS-related stuff (like class) in the template or somewhere close-by is more straightforward.